### PR TITLE
Use document options instead of workspace ones

### PR DESCRIPTION
**Pull request by Pilchie**

### CODE REVIEW

**Comment by Pilchie** [comment]
Tagging @dotnet/roslyn-ide @CyrusNajmabadi @jasonmalinowski

File: src/Features/CSharp/Portable/UseExpressionBody/AbstractUseExpressionBodyCodeFixProvider.cs [hunk 1]
```diff
@@ -41,35 +41,31 @@ internal abstract partial class AbstractUseExpressionBodyCodeFixProvider<TDeclar
             _useBlockBodyTitle = useBlockBodyTitle;
         }
 
-        public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
+        public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             var diagnostic = context.Diagnostics.First();
-            var option = context.Document.Project.Solution.Workspace.Options.GetOption(_option);
+            var option = (await context.Document.GetOptionsAsync(context.CancellationToken).ConfigureAwait(false)).GetOption(_option);
```
Comment (CyrusNajmabadi) [inline src/Features/CSharp/Portable/UseExpressionBody/AbstractUseExpressionBodyCodeFixProvider.cs]: can you break into separate lines.  i really dislike nested awaits in code.

**Comment by CyrusNajmabadi** [comment]
👍

File: src/Features/CSharp/Portable/UseExpressionBody/AbstractUseExpressionBodyCodeFixProvider.cs (hunk 1 above)
Comment (DustinCampbell) [inline src/Features/CSharp/Portable/UseExpressionBody/AbstractUseExpressionBodyCodeFixProvider.cs]: Agreed. Write prettier code @Pilchie!

Comment (jasonmalinowski) [inline src/Features/CSharp/Portable/UseExpressionBody/AbstractUseExpressionBodyCodeFixProvider.cs]: Two lines?

**Pull request merged**